packages/long-short-pair/abis/ERC20.json

```diff
@@ -0,0 +1,288 @@
+[
```

Comment:
We should longer term probably make `abis` its own package and share them between packages. Same goes with a bunch of other files like `utils/Decimals.ts`.

Reply:
agreed